horace_core/configuration/@hpc_config/hpc_config.m

```diff
@@ -219,6 +219,8 @@
             end
             if use_mpi
                 pc = parallel_config;
+                pc.saveable = false;
```

Comment:
is this `pc.saveable=false` required? you needed to remove it elsewhere to propagate configuration to the workers.

Reply:
its safe to do it in local tests which change configuration but not deploy parallel job.

Reply:
This is a global change which will affect tests running MPI jobs too